Tests/NIOTests/EchoServerClientTest.swift

```diff
@@ -653,7 +653,8 @@ class EchoServerClientTest : XCTestCase {
             }.bind(host: "127.0.0.1", port: 0).wait()
 
         defer {
-            _ = serverChannel.close()
+            // Ignore errors here, the channel may already be closed.
+            _ = try? serverChannel.close().wait()
```

Comment:
you should do `serverChannel.syncCloseAcceptingDoubleClose()` or something that we have in the tests

Reply:
Done